PolyGene/codes/ppi_string_RWR.R

```diff
@@ -36,13 +39,30 @@ ppi_string_RWR = function(gene_scores,
 
 
   #############################  Create a String database graph ##############################################
-  string_db <- STRINGdb$new( version="11", species=9606,
```

Comment:
I was having a lot of trouble with STRINGdb, so I changed this to read from a flat file that can be downloaded from the STRING website here: https://string-db.org/cgi/download?sessionId=bm0OFYHgj9Gb&species_text=Homo+sapiens